packages/kbn-ui-shared-deps-src/src/polyfills.js

```diff
@@ -17,3 +17,45 @@ if (typeof window.Event === 'object') {
 
 require('whatwg-fetch');
 require('symbol-observable');
+
+/**
+ * @description Adapted from https://github.com/jantimon/remote-web-worker/blob/main/src/index.ts
```

Comment:
Can we make this an `@notice` so it gets picked up in LICENSES.txt

https://github.com/elastic/kibana/blob/cfbc45615e237791a169e2684342af3e87cc7093/packages/kbn-flot-charts/index.js#L9 is an example.

Reply:
Thanks for the pointer!